transformations/transformations.js

```diff
@@ -5,7 +5,10 @@ const constants = require('../constants');
 
 const transformation = {
   hero_id: ({ field }) => constants.heroes[field],
-  radiant_win: ({ field, match }) => (isRadiant({ playerSlot: match.player_slot }) === field ? 'W' : 'L'),
+  radiant_win: ({ field, match }) => {
+    const isRadiantResult = isRadiant(match.player_slot);
+    return ((isRadiantResult && field) || (!isRadiantResult && !field) ? 'W' : 'L');
```

Comment:
If !field, the match is "no stats recorded".  Up to you if you want to make a specific case for this.